tests/Controller/HelperControllerTest.php

```diff
@@ -89,6 +86,11 @@ public function getEnabled()
     }
 }
 
+interface ModelManagerTest extends ModelManagerInterface
```

Comment:
@greg0ire wdyt about this? We have a problem with the modelManagerInterface... It does not contain getMetadata method, even though all ModelManager from our persistence bundles implements it.

It was removed here: https://github.com/sonata-project/SonataAdminBundle/pull/736

I don't know any other way to fix the initial issue.

Reply:
Fixing the initial issue would be possible by introducing an interface very much like this one and have all model managers implement it in addition to the original one. None of this would be a BC break. The only issue would be finding a good name.